docs/pages/deploy-a-cluster/teleport-enterprise/getting-started.mdx

```diff
@@ -1,465 +1,489 @@
 
 ```code
-# run this on node server:
+$ sudo systemctl enable teleport
 $ sudo systemctl start teleport
-# Confirm node has started successfully and connected
+```
+
+Confirm that the `teleport` service has started: 
+
+```
 $ sudo systemctl status teleport
 ```
 
-## Teleport Status and Networking
+### Get information about your Teleport deployment
 
-You can review the logs of the Teleport service with `journalctl -fu teleport` and see its current status
-with `sudo systemctl status teleport`.
+You can review the logs of the Teleport service with the following command:
 
-You can use `netstat -lptne` to review the port that Teleport is
-listening on  on [TCP/IP ports](../../reference/networking.mdx#ports). On *auth.example.com*, it should
-look something like this:
+```code
+$ journalctl -fu teleport
+```
+Run the following command to review the ports that Teleport is
+listening on:
 
 ```code
-$ auth.example.com ~: sudo netstat -lptne
-Active Internet connections (only servers)
-Proto Recv-Q Send-Q Local Address   State       User       PID/Program name
-tcp6       0      0 :::3024         LISTEN      0          337/teleport
-tcp6       0      0 :::3025         LISTEN      0          337/teleport
-tcp6       0      0 :::3080         LISTEN      0          337/teleport
-tcp6       0      0 :::3022         LISTEN      0          337/teleport
-tcp6       0      0 :::3023         LISTEN      0          337/teleport
+$ sudo netstat -lptne
 ```
 
-and *node.example.com* should look something like this:
+The output should look something like this:
 
 ```code
-$ node.example.com ~: sudo netstat -lptne
-Active Internet connections (only servers)
-Proto Recv-Q Send-Q Local Address   State       User       PID/Program name
-tcp6       0      0 :::3022         LISTEN      0          337/teleport
+$ sudo netstat -lptne | grep teleport
+tcp6       0      0 :::443                  :::*                    LISTEN
+0          168760     29504/teleport    
+tcp6       0      0 :::3022                 :::*                    LISTEN
+0          167812     29504/teleport    
+tcp6       0      0 :::3025                 :::*                    LISTEN
+0          168741     29504/teleport
 ```
 
-See [troubleshooting](#troubleshooting) section at the bottom if something is not working.
-
-## Adding Users
+## Step 4/5. Add a local user
 
-This portion of the Quick Start Guide should be performed on the auth server, i.e.
-on *auth.example.com*
+### Create a user
 
 Every user in a Teleport cluster must be assigned at least one role. By default,
-Teleport comes with several pre-configured roles known as
-["presets"](../../access-controls/reference.mdx#preset-roles). You can see
-these roles by executing `sudo tctl get roles`.
+Teleport comes with several pre-configured roles:
 
-Pay attention to the *allow/logins* field in the role definition: by default, this
-role only allows SSH logins as `root@host`.
```

Comment:
I think we should still find a way to mention the term "presets" when referring to these roles, if for nothing else than to establish consistent vocabulary and maybe even help with SSO.